### DISCUSSION TITLE
Mobbing Resources

### DISCUSSION THREAD

**Discussion by mrozbarry** [mobtimeapp/mobtime#176] [General]
Please add any books, videos, courses, websites, or anything else that you find to be a good mob/jam/ensemble programming resource.

**Comment by mrozbarry** [comment]
[Software Profession Resources trello board](https://trello.com/b/1lfMkCOh/software-profession-resources)

**Comment by jlangr** [comment]
Some of these probably duplicates from the trello board.

https://github.com/remotemobprogramming/mob - [Mob Tool](https://github.com/remotemobprogramming/mob)

https://github.com/remotemobprogramming/upcoming-sessions - [Upcoming open mob sessions](https://github.com/remotemobprogramming/upcoming-sessions)

#OpenPubMobUS, #OpenPubMobUK - Twitter hashtags for weekly, open mob sessions

articles I've written or contributed to:
https://www.ranorex.com/blog/two-rules-mobbing-success/ - [Two Rules for Mobbing Success](https://www.ranorex.com/blog/two-rules-mobbing-success/)
https://www.ranorex.com/blog/does-mobbing-work/ - [Does Mobbing Really Speed Us Up?](https://www.ranorex.com/blog/does-mobbing-work/)
https://www.ranorex.com/blog/need-least-three-sets-eyes-code/ - [Why You Need At Least Three Sets of Eyes on Your Code](https://www.ranorex.com/blog/need-least-three-sets-eyes-code/)
https://blog.gurock.com/mobbing-pairing-solo-development/ - [Balancing Mobbing, Pairing and Solo Work](https://blog.gurock.com/mobbing-pairing-solo-development/)
https://langrsoft.com/2020/06/02/git-handover/ - [Remote collaborative coding: 6 ways to go](https://langrsoft.com/2020/06/02/git-handover/)
https://www.infoq.com/news/2019/02/effective-mob-programming/ - [Effective Mob Programming Patterns](https://www.infoq.com/news/2019/02/effective-mob-programming/)

http://pubmob.com - [PubMob: Remote collaborative software development / training](http://pubmob.com)

Other articles:
https://anthonysciamanna.com/2018/06/29/practices-for-effective-mob-programming.html
https://medium.com/broadlume-product/make-collaboration-consistent-the-case-for-mob-programming-99fffe0e7e91
https://philippe.bourgau.net/best-open-source-tools-for-remote-pair-programming/

Global liquid mob:
https://global-liquid-mob.github.io

A favorite, the (? maybe) original time lapse of a mob programming day.
https://www.youtube.com/watch?v=p_pvslS4gEI
